### PR TITLE
[Merged by Bors] - api: v2alpha1: Add labels_per_unit to NetworkService.Info endpoint

### PR DESCRIPTION
**Pull request by kacpersaw**
https://github.com/spacemeshos/api/pull/365

### CODE REVIEW

File: node/node.go [hunk 1]
```diff
@@ -1559,7 +1559,8 @@ func (app *App) grpcService(svc grpcserver.Service, lg log.Log) (grpcserver.Serv
 		service := v2alpha1.NewNetworkService(
 			app.clock.GenesisTime(),
 			app.Config.Genesis.GenesisID(),
-			app.Config.LayerDuration)
+			app.Config.LayerDuration,
+			app.postSupervisor.Config().LabelsPerUnit)
```
Comment (poszu) [inline node/node.go]: How about
```suggestion
			app.Config.POST.LabelsPerUnit)
```

File: node/node.go [hunk 2]
```diff
@@ -1559,7 +1559,8 @@ func (app *App) grpcService(svc grpcserver.Service, lg log.Log) (grpcserver.Serv
 		service := v2alpha1.NewNetworkService(
 			app.clock.GenesisTime(),
 			app.Config.Genesis.GenesisID(),
-			app.Config.LayerDuration)
+			app.Config.LayerDuration,
+			app.Config.POST.LabelsPerUnit)
```
Comment (fasmat) [inline node/node.go]: Would it make sense to just pass the whole `app.Config` object instead of just 3 fields from it?

**Review by fasmat — APPROVED** [review]

**Comment by kacpersaw** [comment]
bors merge

**Comment by kacpersaw** [comment]
bors merge

**Comment by fasmat** [comment]
bors merge

**Comment by fasmat** [comment]
bors merge

**Pull request closed without merging**